src/components/compose.jsx

```diff
@@ -1178,6 +1178,11 @@ function Compose({
                 <option value="direct">
                   <Trans>Private mention</Trans>
                 </option>
+                {supports('@pleroma/local-only') &&
```

Comment:
This needs to be `@pleroma/local-visibility-post` 😉

Reply:
Ahhhhhhh hahahaha

```suggestion
                {supports('@pleroma/local-visibility-post') &&
```